app/src/main/java/org/schabi/newpipe/player/ui/VideoPlayerUi.java

```diff
@@ -602,9 +602,6 @@ public void onStartTrackingTouch(final SeekBar seekBar) {
         }
 
         player.saveWasPlaying();
```

Comment:
I think this should be removed, too, but I have no idea. Can someone confirm?

Reply:
Yes, we can delete it. I initially thought that this was necessary for the `showControlsThenHide()` method, but it turns out that the check for `wasPlaying` is unnecessary because the visibility of the controls is set in the `onPause()` method separately. After realizing that the `wasPlaying()` method is only used in the `onStopTrackingTouch()` method and is unnecessary there, I removed the `wasPlaying` variable (and its getters and setters) altogether now.